src/Buerokratt.Common/CentOps/CentOpsService.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Buerokratt.Common.CentOps.Interfaces;
+using Buerokratt.Common.CentOps.Models;
+using System.Collections.Concurrent;
+
+namespace Buerokratt.Common.CentOps
+{
+    public class CentOpsService : ICentOpsService
+    {
+        private readonly ConcurrentDictionary<string, Participant> participants;
```

Comment:
nit: Could you rename this to `_participants` to keep it consistent with the naming in the project